extensions/objects/templates/ImageWithRegionsEditor.js

```diff
@@ -233,6 +290,14 @@ oppia.directive('imageWithRegionsEditor', [
           }
           $scope.userIsCurrentlyDrawing = false;
           $scope.userIsCurrentlyDragging = false;
+          $scope.userIsCurrentlyResizing = false;
+          if ($scope.outOfRegion) {
+            $scope.xDirection = 0;
+            $scope.yDirection = 0;
+          }
```

Comment:
I've noticed a subtle inconsistency with this. When resizing "out of region" the xDirection and yDirection will be set to 0, and so the cursor will be set to default. However, when resizing within region the cursor will not be affected. You could just make it so that they're always set to 0 regardless and get rid of the outOfRegion variable. Otherwise, you'd want to make it so that the cursor isn't changed on mouse up in both cases. Either is fine I think, just as long as the behavior is consistent.